server/config/cloudinaryConfig.js

```diff
@@ -0,0 +1,9 @@
+const cloudinary = require("cloudinary").v2;
```

Comment:
<div>


<div id="suggestion">
<div id="issue"><b>Consider using ES6 import statement</b></div>
<div id="fix">

Consider using ES6 `import` statement instead of `require()`. This helps maintain consistency with modern JavaScript practices and improves code maintainability. You can update the import statement and export declaration to use ES6 syntax.
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<blockquote>Check the AI-generated fix before applying</blockquote>
<div id="code">


```
 @@ -1,9 +1,9 @@
 -const cloudinary = require("cloudinary").v2;
 -
 -cloudinary.config({
 +import { v2 as cloudinary } from 'cloudinary';
 +
 cloudinary.config({
    cloud_name: process.env.CLOUD_NAME,
    api_key: process.env.API_KEY,
    api_secret: process.env.API_SECRET,
  });
 
 -module.exports = cloudinary;
 +export default cloudinary;
```

</div>
</details>
</div>


<small><i>Code Review Run <a href=https://github.com/Divyanshu9822/blog-mern-app/pull/22#issuecomment-2544587479>#245782</a></i></small>
</div>

---
Is this a valid issue, or was it incorrectly flagged by the Agent?
- [ ] it was incorrectly flagged